internal/common/vote/vote.go

```diff
@@ -0,0 +1,191 @@
+package vote
+
+import (
+	"math/rand"
+	"time"
+)
+
+func Get_votes_for_rule(ruleID string, islandID int) map[int][]int {
+	votes_layout := make(map[int][]int)
+	rand.Seed(time.Now().UTC().UnixNano())
+	i := 1
+	for {
+		votes_layout[i] = []int{rand.Intn(100), rand.Intn(100)}
+		i++
+		if i > 6 {
+			break
+		}
+	}
```

Comment:
```
votesLayout = make(map[int][]int)
for i := 1; i <= 6; i++ {
    votesLayout[i] = []int{rand.Intn(100), rand.Intn(100)}
}
```

Reply:
Don't hardcode the number of islands, we may want to have more than one island per team in the future (e.g. same client code initialized with different values). This should be resolved throughout the PR